### DISCUSSION TITLE
Pending Support Ticket

### DISCUSSION THREAD

**Discussion by ColinWater** [community/community#23079] [New to GitHub]
<p>Hello there, my ticket status still pending for 2 weeks now. Please someone help me out. Thanks.</p>
<p>Ticket ID 1673432</p>

**Comment by queenofcorgis** [comment, marked as answer]
<p>Hi there! I see this was solved - thanks for your patience :smile:. For additional information, Support tickets are answered in the order they are received and wait times may vary based on the tickets ahead of you and staff resources.</p>
<p>The GitHub Community Forum is unable to escalate any tickets.</p>
<p>Glad to see this was solved!</p>

**Reply by anasty17** [reply]
Same here ticket 2134810

**Comment by steve3140** [comment]
One wonders what then, if anything, can be done.
Okay sure, I'm not an Enterprise customer, but I am still a paying customer.
Logged my ticket #2110988 on 15 APR so that's basically 3 weeks with zero response.
